typings/index.d.ts

```diff
@@ -887,7 +887,7 @@ declare module 'discord.js' {
     constructor(client: Client, data: object);
     public approximateMemberCount: number;
     public approximatePresenceCount: number;
-    public description?: string;
+    public description?: string | null;
```

Comment:
It appears to be `null` when there's no description, instead of `undefined`.

```suggestion
    public description: string | null;
```